docs/getting-started/tutorial-sample-edit.asciidoc

```diff
@@ -0,0 +1,45 @@
+[[tutorial-sample-edit]]
+=== Editing a visualization
+
+You have edit permissions for the *Global Flight* dashboard so you can change 
+the appearance and behavior of the visualizations. For example, you might want 
+to see which airline has the lowest average fares.
+
+. Return to the *Global Flight* dashboard. 
+. In the menu bar, click *Edit*.
+. In the *Average Ticket Price* visualization, click the gear icon in 
```

Comment:
I would try to be super explicit here by providing the exact full string of the visualization `[Flights] Average Ticket ` this way the user wouldn't be confused with the other visualization which also is similar `[Flights] Flight Count and Average Ticket Price`